### PR TITLE
Created to burn time

### LINKED ISSUE #7: baseline dominates issues ....

### PR DESCRIPTION
**Pull request by UweSchweickert**
this pr resolves #7

### CODE REVIEW

File: hometown/test.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## places to eat
+
+## things to do
+
+## gin tasting
```
Comment (UweSchweickert) [inline hometown/test.md]:
```suggestion
## gin tasting
```

Comment (slohmi) [inline hometown/test.md]: verwirrt

File: hometown/test.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## places to eat
+
```
Comment (hubertkret) [inline hometown/test.md]:
```suggestion
What about the Medici restaurant?
```

File: hometown/test.md (hunk 1 above)
Comment (UweSchweickert) [reply hometown/test.md]: Hunger

😋

**Pull request merged**